### PR TITLE
Makefile

### PR DESCRIPTION
**Pull request by ivanterekh**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+APP ?= go-skeleton
+
+build:
```
Comment (dzyanis) [inline Makefile]: I'd recommend to addd `.PHONY`

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+	@GO111MODULE=on go build -o ${APP}
+
+run: build
```
Comment (dzyanis) [inline Makefile:8]: Why don't we use `go vet` and `go impoort` before `build`

**Review by dzyanis — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (ivanterekh) [reply Makefile]: Done.

File: Makefile (hunk 2 above)
Comment (ivanterekh) [reply Makefile:8]: Done.

**Pull request merged**